<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetreParametresRecherche</class>
 <widget class="QDialog" name="FenetreParametresRecherche">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paramètres de recherche</string>
  </property>
  <layout class="QGridLayout" name="gridLayoutFenetreParametresRecherche">
   <item row="0" column="0">
    <widget class="QLabel" name="labelSeuilToleranceNiveauxDeGris">
     <property name="text">
      <string>Seuil de tolérance des niveaux de gris</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxSeuilToleranceNiveauxDeGris">
     <property name="decimals">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelSeuilToleranceTeintesSaturees">
     <property name="text">
      <string>Seuil de tolérance des teintes saturées</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxSeuilToleranceTeintesSaturees">
     <property name="decimals">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelSelectionValeursMoyennes">
     <property name="text">
      <string>Sélection des valeurs moyennes</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="checkBoxSelectionValeursMoyennes">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelSelectionValeursMinimales">
     <property name="text">
      <string>Sélection des valeurs minimales</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="checkBoxSelectionValeursMinimales">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelSelectionValeursMaximales">
     <property name="text">
      <string>Sélection des valeurs maximales</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="checkBoxSelectionValeursMaximales">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FenetreParametresRecherche</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FenetreParametresRecherche</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
